lib/src/contracts/contract_invoker.dart

```diff
@@ -0,0 +1,118 @@
+part of 'package:web3dart/contracts.dart';
+
+class ContractInvocation {
+  final Web3Client client;
+  final DeployedContract contract;
+  final ContractFunction function;
+  List _params;
+
+  ContractInvocation(this.client, this.contract, this.function);
+
+  ContractInvocation parameters([dynamic args]) {
```

Comment:
I would suggest this documentation:
```suggestion
  /// Creates a new [ContractInvocation] instance that will call the function with provided [args].
  ///
  /// Args can either be:
  ///  - a `Map`, where the key is the parameter name and the value is the argument value
  //// - a `List`, which behaves like [ContractFunction.encodeCall].
  ////  -a simple value, which will be wrapped in a `List` and used as a single argument.
  ContractInvocation parameters([dynamic args]) {
```